cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -40,7 +40,7 @@
                    class="m-form-field-with-button_field"
                     {{ "required" if value.required else "" }}>
         </div>
-        {{ value.info or '' }}
+        {{ value.info | safe }}
```

Comment:
hmm yea it could be.. i just didn't know if there was a purpose for outputting an empty string